src/main/java/io/stargate/sgv2/jsonapi/exception/DebugModeConfig.java

```diff
@@ -0,0 +1,11 @@
+package io.stargate.sgv2.jsonapi.exception;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "stargate.debug")
+public interface DebugModeConfig {
+
+  @WithDefault("false")
+  boolean enable();
```

Comment:
We may want to rename this as `enabled` to be compatible with other settings; I'll change.

Reply:
Renamed.